mangaki/mangaki/admin.py

```diff
@@ -76,6 +191,10 @@ class WorkAdmin(admin.ModelAdmin):
         'controversy',
     )
 
+    def get_queryset(self, request):
+        qs = super().get_queryset(request)
+        return qs.filter(redirect__isnull=True)  # Exclude phantom works
```

Comment:
Yes but I can't make it. Can you? I mean, how to add extra elements when the whole source is filtered?